proof/courier.go

```diff
@@ -578,6 +584,27 @@ type BackoffHandler struct {
 func (b *BackoffHandler) initialDelay(ctx context.Context,
 	proofLocator Locator, proofTransferType TransferType) error {
 
+	// If there is an initial delay set, we'll always wait for that amount
+	// of time, regardless of what the transfer log contains. This is to
+	// give the sender some time to transfer the proof to the universe if
+	// a universe RPC courier is being used.
```

Comment:
I think the same delay is technically relevant to the hashmail courier as well. I'm not sure.